src/app/src/main/java/com/cmput301f21t09/budgetprojectname/views/activities/DefineHabitEventActivity.java

```diff
@@ -211,15 +219,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
                     HabitEventModel habitEvent = new HabitEventModel(null,
                             locationSwitch.isChecked() && (markerLocation.getLatitude() != null) ? markerLocation : null,
                             new Date(), commentStr, null, habitID);
-
                     if (isNewHabitEvent) {
                         habitEventController.createHabitEvent(habitEvent, new HabitEventController.HabitEventIDCallback() {
                             @Override
                             public void onCallback(String habitEventID) {
                                 // TODO: figure out what to add here
```

Comment:
is this TODO statement no longer needed?

Reply:
I didn't add it in the first place, so I'm not sure if whoever added it is done with it. I believe @MacMalainey added it

Reply:
Never have edited this file.

Reply:
That todo can be deleted